source/MaterialXRuntime/Private/Commands/PvtBreakConnectionCmd.cpp

```diff
@@ -19,7 +19,12 @@ void PvtBreakConnectionCmd::execute(RtCommandResult& result)
     {
         try
         {
+            // Send message that the connection is to be broken
+            msg().sendBreakConnectionMessage(_src, _dest);
```

Comment:
The message is sent before the action has taken place. If a poorly designed UI starts browsing the stage at the moment it receives the notification, then it will not observe the final state. Maybe send the notifications after?

Reply:
Ah yes, since we track both connection made and connection broken it's better to send the message after the fact. I will change that.